src/attestation/sev.rs

```diff
@@ -84,46 +97,97 @@ mod tests {
 
     use super::*;
 
+    fn test_message_representation(bytes: &[u8], expected: Message) {
+        let actual: Message = serde_flavor::from_reader(bytes).unwrap();
```

Comment:
Where is `serde_flavor` implemented / documented? Having trouble finding it.

Reply:
[`serde_flavor` is a type alias for `serde_cbor`](https://github.com/enarx/koine/pull/11/files#diff-8b1c3c93abd30a3d72c9a5e2ab59201a8a0cdc81f5b1e0029b41b5c946d39a26R96).

Reply:
Ah, gotcha. Your link is not working for me, but that makes a lot of sense.

Reply:
Oh, I see, it's right here: https://github.com/connorkuehl/koine/blob/need-more-cbor/src/attestation/sev.rs#L96